dts/msm8974/msm8974pro-sony-xperia-shinano-leo.dts

```diff
@@ -0,0 +1,1356 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	model = "Sony Xperia Z3";
+	compatible = "sony,xperia-leo", "qcom,msm8974", "lk2nd,device";
+	qcom,msm-id = <194 8 0x10000>;
+	qcom,board-id = <8 0>;
+
+	lk2nd,keys =
+		<KEY_VOLUMEUP   PM_GPIO(5) PM_GPIO_PULL_UP_1_5>,
+		<KEY_VOLUMEDOWN PM_GPIO(2) PM_GPIO_PULL_UP_1_5>;
+
+	soc {
```

Comment:
I think we usually move things like this into a dedicated .dtsi
like this: https://github.com/msm8916-mainline/lk2nd/blob/master/dts/msm8916/motorola-carrier-channel-ids.dtsi